apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md

```diff
@@ -0,0 +1,280 @@
+        SAMPLE_SYNTHEA_BENEINTERIM(
+        resourceUrl("rif-synthea/beneficiary_interim.csv"), RifFileType.BENEFICIARY, 10000),
+        SAMPLE_SYNTHEA_BENEFINAL(
+        resourceUrl("rif-synthea/beneficiary_final.csv"), RifFileType.BENEFICIARY, 10000),
+        SAMPLE_SYNTHEA_CARRIER(resourceUrl("rif-synthea/carrier.csv"), RifFileType.CARRIER, 279900),
+        SAMPLE_SYNTHEA_INPATIENT(resourceUrl("rif-synthea/inpatient.csv"), RifFileType.INPATIENT, 36606),
+        SAMPLE_SYNTHEA_OUTPATIENT(
+        resourceUrl("rif-synthea/outpatient.csv"), RifFileType.OUTPATIENT, 328420),
+        SAMPLE_SYNTHEA_SNF(resourceUrl("rif-synthea/snf.csv"), RifFileType.SNF, 2797),
+        SAMPLE_SYNTHEA_HOSPICE(resourceUrl("rif-synthea/hospice.csv"), RifFileType.HOSPICE, 1396),
+        SAMPLE_SYNTHEA_HHA(resourceUrl("rif-synthea/hha.csv"), RifFileType.HHA, 14377),
+        SAMPLE_SYNTHEA_DME(resourceUrl("rif-synthea/dme.csv"), RifFileType.DME, 8727),
+        SAMPLE_SYNTHEA_PDE(resourceUrl("rif-synthea/pde.csv"), RifFileType.PDE, 214157),
+        SAMPLE_SYNTHEA_BENEHISTORY(
+        resourceUrl("rif-synthea/beneficiary_history.csv"), RifFileType.BENEFICIARY_HISTORY, 10000),
+
+3. Go to `/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java` and navigate to where the SYNTHEA_DATA type is and make sure the entries from Step 2 are represented underneath the type
+
+StaticResourceGroup:
+
+    SYNTHEA_DATA(
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2011,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2012,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2013,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2014,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2015,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2016,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2017,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2018,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2019,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2020,
+        StaticRifResource.SAMPLE_SYNTHEA_BENES2021,
+        StaticRifResource.SAMPLE_SYNTHEA_CARRIER,
+        StaticRifResource.SAMPLE_SYNTHEA_INPATIENT,
+        StaticRifResource.SAMPLE_SYNTHEA_OUTPATIENT,
+        StaticRifResource.SAMPLE_SYNTHEA_SNF,
+        StaticRifResource.SAMPLE_SYNTHEA_HOSPICE,
+        StaticRifResource.SAMPLE_SYNTHEA_HHA,
+        StaticRifResource.SAMPLE_SYNTHEA_DME,
+        StaticRifResource.SAMPLE_SYNTHEA_PDE,
+        StaticRifResource.SAMPLE_SYNTHEA_BENEHISTORY),
+
+4. Go to `/apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java` and comment out the @Ignore from the loadSyntheaData() test.
+
+5.  Run the following in a terminal window: 
+`Docker
+docker run \
+     -d \
+     --name 'bfd-db' \
+     -e 'POSTGRES_DB=bfd' \
+     -e 'POSTGRES_USER=bfd' \
+     -e 'POSTGRES_PASSWORD=InsecureLocalDev' \
+     -p '5432:5432' \
+     -v 'bfd_pgdata:/var/lib/postgresql/data' \
+     postgres:12`
```

Comment:
if we reference this we should probably also include a step to clean up any previous docker (so that there is no hold over data):

docker stop bfd-db
docker rm bfd-db
docker volume rm bfd_pgdata

( I didn't test these but I think they are close to correct)